### DISCUSSION TITLE
Welcome to Lightbi Discussions!

### DISCUSSION THREAD

**Discussion by binokochumolvarghese** [binokochumolvarghese/lightbi-hugo#15] [Announcements]
Welcome 👋

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions about things you’re wondering about.
- Share ideas for new features or enhancements.
- Show off what you've built using Lightbi.
- Engage thoughtfully with other community members.

Need to report a bug?
🐛 If you have found a bug or other issue, don't post about it here. [Open an issue](https://github.com/binokochumolvarghese/lightbi-hugo/issues/new) instead.

**Comment by ILIG-MESSINA** [comment]
Hello Team, Please i'm trying to build a website based on Lightbi template on Rstudio but the menu is not appearing i don't know why ! Can someone help please ?

**Comment by binokochumolvarghese** [comment]
Please add menu in your config.toml file as below:
```

[[menu.main]]
    name = "Home"
    url = "/"
    weight = 1
[[menu.main]]
    name = "Blog"
    url = "/blog/"
    weight = 2

```

**Comment by mialunachen** [comment]
Hi, what does "copy the contents of exampleSite to the content folder in your website" mean? Does it just mean to copy the content folder of exampleSite into the content folder in your website, or to copy the entire exampleSite into the content folder? Thanks

**Reply by binokochumolvarghese** [reply]
Hi, just copy the content folder of exampleSite into the content folder in your website. 

You can find the demo website [here](https://github.com/binokochumolvarghese/lightbi-hugo-demo) and more detailed documentation [here](https://binovarghese.com/blog/2021/build-your-first-hugo-blog/).

**Reply by mialunachen** [reply]
My "tags" page is getting a 404 error, any idea of why this is happening?

**Reply by binokochumolvarghese** [reply]
I am sorry, I cann't comment without seeing the code.

You can check the demo [here](https://github.com/binokochumolvarghese/lightbi-hugo-demo) of live website.

**Comment by mialunachen** [comment]
The dates in my blog are displaying very weirdly, for example instead of "27 January 2020" it displays "27 January 272712" 
Any idea of what could be causing this issue and where I can diagnose the root of this problem?

**Reply by binokochumolvarghese** [reply]
Check the below is added or not in the config file:

```
[Params]
dateFormat = "2 January 2023"
```

Hope this help you.

**Comment by hu921** [comment]
Hello Team, Please i'm trying to build a website based on Lightbi template, I'm facing two issues here. can someone help?
1. I can't get Favicon work, I add the icons under /static/favicon/
2. I added image under assets, only image can be shown in about section, but not in the post .
Here's my code "![image1!](/images/stock_0.jpg "image1")
Thank you so much.

**Reply by binokochumolvarghese** [reply]
I can't see you r code. Please update the same. 

You can check the demo [here](https://github.com/binokochumolvarghese/lightbi-hugo-demo) of a live website to verify the code.
